client/src/document/ingredients/browser-compatibility-table/browser-info.tsx

```diff
@@ -0,0 +1,14 @@
+import React, { useContext } from "react";
+import type bcd from "mdn-browser-compat-data/types";
+
+export const BrowserInfoContext = React.createContext<bcd.Browsers | null>(
+  null
+);
+
+export function BrowserName({ id }: { id: bcd.BrowserNames }) {
+  const browserInfo = useContext(BrowserInfoContext);
+  if (!browserInfo) {
+    throw new Error("Missing browser info");
+  }
+  return <>{browserInfo[id].name}</>;
```

Comment:
Why not just the name itself? Like 
```suggestion
  return browserInfo[id].name
```